src/node-helper.js

```diff
@@ -31,12 +42,26 @@ const validateContentstackAccess = async pluginOptions => {
   let host = pluginOptions.cdn
     ? pluginOptions.cdn
     : 'https://cdn.contentstack.io/v3';
+
+  let headers = {
+    api_key: `${pluginOptions.api_key}`,
+    access_token: `${pluginOptions.delivery_token}`,
+    branch: pluginOptions?.branch,
+  };
+  // Check if config has the key: enableEarlyAccess, it's an array of strings, and it's not empty
+  if (
+    pluginOptions?.enableEarlyAccessValue &&
+    pluginOptions?.enableEarlyAccessKey &&
+    pluginOptions?.enableEarlyAccessKey !== '' &&
+    Array.isArray(pluginOptions.enableEarlyAccessValue) &&
+    pluginOptions.enableEarlyAccessValue.length > 0
+  ) {
+    const earlyAccessHeaders = pluginOptions.enableEarlyAccessValue.join(',');
+    headers[pluginOptions?.enableEarlyAccessKey] = earlyAccessHeaders;
+  }
+
   await fetch(`${host}/content_types?include_count=false`, {
-    headers: {
-      api_key: `${pluginOptions.api_key}`,
-      access_token: `${pluginOptions.delivery_token}`,
-      branch: pluginOptions?.branch,
-    },
+    headers: headers,
```

Comment:
rename this

Reply:
remove this way of implementation. hence resolving this.